tgstation.dme

```diff
@@ -6,6 +6,80 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "_maps"
```

Comment:
https://tgstation13.org/phpBB/viewtopic.php?f=5&t=321

Just going to pop this here